Bump the PHP version used in the CI

greg0ire opened the pull request:
8.4 is stable now.

greg0ire committed:
Bump the PHP version used in the CI

8.4 is stable now.

morozov commented:
Should we maybe stop explicitly testing PHP 8.2? If the code works on 7.4 and 8.4, most likely it will work on all version in between.

There's also a single PHP 8.1 SQLite build. It doesn't look like it covers any dedicated execution path.

greg0ire committed:
Remove jobs for PHP 8.2

If the code works on 7.4 and 8.4, most likely it will work on all versions in between.

greg0ire committed:
Remove lone 8.1 build

Not sure what this one was about.

morozov commented:
It looks like there's annual/weekend IBM driver hosting maintenance going on. Let's ignore those failures for now.
